src/main/java/org/commcare/formplayer/db/migration/V22__init_entities_selection.java

```diff
@@ -0,0 +1,19 @@
+package org.commcare.formplayer.db.migration;
+
+import java.util.Arrays;
+
+public class V22__init_entities_selection extends BaseFormplayerMigration {
+
+    @Override
+    public Iterable<String> getSqlStatements() {
+        return Arrays.asList(
+                "CREATE TABLE entities_selection (\n" +
+                        "    id text NOT NULL,\n" +
```

Comment:
Since this is auto-generated and will always be a UUID I think the column should be `UUID` type.

Reply:
makes sense. separately do you think we need to store info like `user` , `domain` and `appid` in the table. I see that we do that for `MenuSession` even if they are not used explicity.

Reply:
Good point. We should probably store some information that allows us to verify that the user has access to the selections. So storing the userid / username should be sufficient.

Storing the domain has no functional use right at the present but it could be useful in the future either functionally or else for maintenance / metrics.

Reply:
https://github.com/dimagi/formplayer/pull/1118/commits/39d6a28383256b4aa0172ea52878cec32b19d8e6